cpp/arcticdb/pipeline/column_stats.cpp

```diff
@@ -254,14 +254,14 @@ void ColumnStats::parse_version(std::string_view version_string) {
                 dot_position != std::string::npos,
                 "Unexpected version string in column stats column name (expected vX.Y): {}", version_string);
     auto candidate = version_string.substr(1, dot_position - 1);
-    uint64_t major_version;
+    uint64_t major_version = 0;
     auto result = std::from_chars(candidate.data(), candidate.data() + candidate.size(), major_version);
```

Comment:
Could also switch to `stoull` I guess

Reply:
Indeed. IMO, this better be done in another PR since this intent is orthogonal to the one of this one. What do you think?